tests/logcat-parse-Tests/LogcatParse-Tests.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should we rename this file to `logcat-parse-Tests.csproj` so it matches the directory name?

(How is the output assembly determined, anyway?  From enclosing directory name?  `.csproj` base name?)

Reply:
I can rename it.  I'm pretty sure it's from the `.csproj` name.